#@module_name hsr_navigation
geometry_msgs/Pose2D pose
---
bool success
